lib/BackendClient.js

```diff
@@ -494,6 +496,28 @@ var BackendClient = (function(){
 
     }
 
+    function registerToPush(templateSystemName, partnerIdToRegister, entryId)
```

Comment:
i think we need that to be a bit more generic and not to pass partnerIdToRegister, entryId
maybe it should get an object that you serialze into userParams